### ISSUE TITLE
ULTRA RARE DARK GEMS

### ISSUE THREAD

**Issue by Modeeves** [rubensworks/EvilCraft#923]
#### Issue type:

- :question: Question 

____

#### Issue type:

- :bug: Bug 

____

#### Short description:

I can't find the dark gems

#### Steps to reproduce the problem:

I went down to the caves searching between y50 to y-60 and I can't find any dark gems in the Overworld (of course)

#### Expected behaviour:

Back in 1.16, the dark gems were very frequent and easy to find. But now it isn't.
____

#### Versions:

- This mod: EvilCraft-1.19-1.2.2
- Minecraft: 1.19
- Forge: 46.0.62

**Comment by rubensworks** [comment]
Someone will answer your question soon. In the meantime, you might be able to get help more quickly on our [Discord server](https://discord.gg/9yDxubB).

**Comment by rubensworks** [comment]
Thanks for reporting!

**Comment by rubensworks** [comment]
Are you aware that the texture has changed?
It now looks like this: ![](https://evilcraft.rubensworks.net/book/assets/icons/evilcraft__dark_ore.png)

When debugging, I was able to find them in any case.
But I can check again if you can confirm you were looking for ores matching the texture above.

**Comment by Modeeves** [comment]
Hello, there. Thanks for the kind response. Apparently, I just discovered that this is not just happening on your mod. I tried other mods with modded ores that suppose to generate in the cave but when I look for them, they are also not showing up :-(
I'm not sure what's happening with my Minecraft. I also resorted to removing BOP or BYO or any other modded biomes or caves but they are still not showing up including yours.
